### PR TITLE
Add support for user-defined Albers Equal Area projection CRS to GeoTIFF Writer

### PR DESCRIPTION
**Pull request by jessicaaustin**

### CODE REVIEW

File: raster/src/main/scala/geotrellis/raster/io/geotiff/writer/CoordinateSystemParser.scala [hunk 1]
```diff
@@ -305,6 +306,36 @@ class CoordinateSystemParser(val crs: CRS, val pixelSampleType: Option[PixelSamp
...
+      (ProjCoordTransGeoKey, CT_AlbersEqualArea)
+    )
+
+    val doublesLB = ListBuffer[(Int, Double)]()
+
+    doublesLB += (ProjNatOriginLatGeoKey -> lat0)
+    doublesLB += (ProjNatOriginLongGeoKey -> getDouble("lon_0"))
+
+    if (lat0 == lat1) {
+      doublesLB += (ProjScaleAtNatOriginGeoKey -> getK(1.0))
```
Comment (pomadchin) [inline raster/src/main/scala/geotrellis/raster/io/geotiff/writer/CoordinateSystemParser.scala]: Hm, is here a support of `k` in `aea` projection?

**Comment by pomadchin** [comment]
You can find necessary params in [GDAL](https://github.com/OSGeo/gdal/blob/8d1cde33ca846c3a4071d5b7bf3035f2ad98e2f3/gdal/frmts/gtiff/gt_wkt_srs.cpp#L1567-L1596). It is possible to simplify `aeaProps`:

```scala
private lazy val aeaProps = {
  val geoKeysInt = List(
    (GTModelTypeGeoKey, ModelTypeProjected),
    (ProjectedCSTypeGeoKey, UserDefinedCPV),
    (ProjectionGeoKey, UserDefinedCPV),
    (ProjCoordTransGeoKey, CT_AlbersEqualArea)
  )

  val doubles = List(
    (ProjStdParallel1GeoKey, getDouble("lat_1")),
    (ProjStdParallel2GeoKey, getDouble("lat_2")),
    (ProjNatOriginLatGeoKey, getDouble("lat_0")),
    (ProjNatOriginLongGeoKey, getDouble("lon_0")),
    (ProjFalseEastingGeoKey, getDouble("x_0")),
    (ProjFalseNorthingGeoKey, getDouble("y_0"))
  )

  (geoKeysInt, doubles)
}
```

**Comment by pomadchin** [comment]
Instructions to sign up Eclipse CLA (if you don't have one): [CONTRIBUTING.rst](https://github.com/locationtech/geotrellis/blob/master/CONTRIBUTING.rst#eclipse-contributor-agreement-eca)

**Comment by jessicaaustin** [comment]
Yes, that's much simpler! I've made the changes, and fixed the IP.

I had issues getting all the tests in the `raster` suite running, but I was able to run `./sbt "test:testOnly *GeoTiffWriterSpec" "project raster"` successfully so I hoped it was something else. Please let me know if there's anything else you need. 

Thank you very much for your help, and for your work on this excellent project!

**Pull request merged**